Hello_Serialization_stud/Hello_Serialization_stud/Program.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Runtime.Serialization.Formatters.Soap;
+using System.Xml.Serialization;
+using System.Runtime.Serialization;
+
+namespace Hello_Serialization_stud
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var student1 = new Student();
+            student1.FirstName = "Marco";
+            student1.LastName = "Polo";
+            student1.Nationality = "Venice";
+            student1.SetAddress("Hreschatyk", "10");
+            Console.WriteLine(student1);
+
+            var student2 = new Student();
+            student2.FirstName = "Cristo";
+            student2.LastName = "Columb";
+            student2.Nationality = "Spanish";
+            student2.SetAddress("Shevchenka", "20");
+            Console.WriteLine(student2);
+
+            var students = new List<Student>();
+            students.Add(student1);
+            students.Add(student2);
+
+            BinarySerialize(students);
+
+            SoapSerialize(students);
+
+            XmlSerialize(student1);
+            XmlSerialize(student2);
+        }
+
+        public static void BinarySerialize(List<Student> students)
+        {
+            var binaryFilePath = "binary.bin";
+            Console.WriteLine("BinarySerialize ......");
+
+            using (Stream streamWrite = new FileStream(binaryFilePath, FileMode.Create, FileAccess.Write, FileShare.None))
+            {
+                IFormatter serializeFormatter = new BinaryFormatter();
+
+                foreach (var student in students)
+                {
+                    serializeFormatter.Serialize(streamWrite, student);
+                }
+
+                streamWrite.Close();
```

Comment:
using should care about this, do i'm right?

Reply:
Yes, you right, I forgot about it.

Reply:
Fixed in 7c15d90c197d568455360facf70394aa09105e5c